src/lib/colors.ts

```diff
@@ -0,0 +1,35 @@
+import { RGBColor } from 'react-color'
+
+export function hexToRgb(hex: string) {
+  const result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hex)
+  return result
+    ? {
+        r: parseInt(result[1], 16),
+        g: parseInt(result[2], 16),
+        b: parseInt(result[3], 16),
+      }
+    : null
```

Comment:
Don't return `null`. If the hex value is invalid, throw an error or return a default value. I guess white color,`rgb(255,255,255)`, make more sense in this case.